src/App.js

```diff
@@ -1,25 +1,58 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
 
 function App() {
+  
+  const [list, setList] = useState(foods);
+
+  const addFood = (food) => {
+    const foodListCopy = [...list, food]
+    setList(foodListCopy)
+
```

Comment:
You need to close this `addFood` function before this return below. Otherwise, it is not reading the values in your `return` correctly.